superset-frontend/plugins/legacy-plugin-chart-country-map/src/CountryMap.js

```diff
@@ -37,7 +37,6 @@ const propTypes = {
   width: PropTypes.number,
   height: PropTypes.number,
   country: PropTypes.string,
-  colorScheme: PropTypes.string,
```

Comment:
Do we not support categorical use cases here? I forgot the details, but I think there was a way to use this visualization to show categorical classification in some cases. Like plotting categorical for something like "most grown crop" on a per-state basis.

Seems the color scheme related changes here don't align with the title/body of the PR (which is about iso code support)